tests/test_topology.py

```diff
@@ -74,7 +76,8 @@ def parameterize_nonbonded_full(
 
 @pytest.mark.parametrize("precision, rtol, atol", [(np.float64, 1e-8, 1e-8), (np.float32, 1e-4, 5e-4)])
 @pytest.mark.parametrize("ctor", [BaseTopology, DualTopology, DualTopologyMinimization])
-def test_host_guest_nonbonded(ctor, precision, rtol, atol):
+@pytest.mark.parametrize("use_tiny_mol", [True, False])
```

Comment:
Coverage: Would there be benefit to testing other small corner cases, 1, 2, 3, 4, 4+ atoms? (so the molecule may have no bonds, angles, torsions?)

Naming / comment: "mol with no intramolecular NB terms" to indicate the challenge presented by this input

Reply:
> Coverage: Would there be benefit to testing other small corner cases, 1, 2, 3, 4, 4+ atoms? (so the molecule may have no bonds, angles, torsions?)

Molecules smaller than 3 atoms have other issues (atom mapping can fail, AM1BCC seems to fail). I'll file a case for this but it's not as high a priority.

> Naming / comment: "mol with no intramolecular NB terms" to indicate the challenge presented by this input

Added comment to tests